doc/source/comparison_with_r.rst

```diff
@@ -31,6 +31,79 @@ For transfer of ``DataFrame`` objects from ``pandas`` to R, one option is to
 use HDF5 files, see :ref:`io.external_compatibility` for an
 example.
 
+
+Quick Reference
+---------------
+
+We'll start off with a quick reference guide pairing some common R
+operations using `dplyr
+<http://cran.r-project.org/web/packages/dplyr/index.html>`__ with
+pandas equivalents.
+
+
+Querying, Filtering, Sampling
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+===========================================  ===========================================
+R                                            pandas
+===========================================  ===========================================
+``dim(df)``                                  ``df.shape``
+``head(df)``                                 ``df.head()``
+``slice(df, 1:10)``                          ``df.iloc[:9]``
+``filter(df, col1 == 1, col2 == 1)``         ``df.query('col1 == 1 & col2 == 1')``
+``df[df$col1 == 1 & df$col2 == 1,]``         ``df[(df.col1 == 1) & (df.col2 == 1)]``
+``select(df, col1, col2)``                   ``df[['col1', 'col2']]``
+``select(df, col1:col3)``                    No one-line equivalent, but see [#select_range]_
```

Comment:
Probably.  I'm not a pandas expert.  I was going off http://nbviewer.jupyter.org/urls/gist.githubusercontent.com/TomAugspurger/6e052140eaa5fdb6e8c0/raw/811585624e843f3f80b9b6fe89e18119d7d2d73c/dplyr_pandas.ipynb which said there wasn't an equivalent.  @TomAugspurger is that correct?